src/pudl/etl/check_foreign_keys.py

```diff
@@ -50,15 +61,147 @@ def pudl_check_fks(logfile: pathlib.Path, loglevel: str):
     # Display logged output from the PUDL package:
     pudl.logging_helpers.configure_root_logger(logfile=logfile, loglevel=loglevel)
 
-    context = build_init_resource_context()
-    io_manager = pudl_sqlite_io_manager(context)
+    # Using PudlPaths to get default value for CLI causes validation issues
+    if not db_path:
+        db_path = PudlPaths().output_dir / "pudl.sqlite"
 
-    database_path = io_manager.base_dir / f"{io_manager.db_name}.sqlite"
-    logger.info(f"Checking foreign key constraints in {database_path}")
```

Comment:
This is logged directly in the `check_foreign_keys` function, so this was just removing a duplicate log

Reply:
Yes, but it only includes the `engine.name` and not the path, which is less informative. See note below.

Reply:
Yes, but it only includes the `engine.name` and not the path, which is less informative. See note below.

Reply:
Ah yeah that makes sense, I've added the log back in.

Reply:
Oh I think having a single logging message is fine, it should just include the path. @rousik notes below that we can use `engine.url` rather than `engine.name`